persistence/jdbc/src/test/java/org/infinispan/persistence/jdbc/TableManagerTest.java

```diff
@@ -128,7 +126,7 @@ public void testDrop() throws Exception {
    }
 
    public void testTableQuoting() throws Exception {
-      tableManager.setCacheName("my.cache");
+      tableManager.dropTable(connection);
```

Comment:
The table manager was setting a cache name that does not exist. The next step, it was checking if the table exists. Instead of `setCacheName` due that the method was removed, I decided to drop the current table.